packages/name-controller/CHANGELOG.md

```diff
@@ -6,9 +6,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+### Changed
+- **BREAKING**: Support rate limiting in name providers ([#1715](https://github.com/MetaMask/core/pull/1715))
```

Comment:
@matthewwalsh0 @vinistevam I wasn't sure how to characterize the change properly, so I've just used the changes listed in the PR description. Please feel free to suggest any improvements you can think of